artifactory/services/upload.go

```diff
@@ -124,10 +177,42 @@ func addSymlinkProps(artifact clientutils.Artifact, uploadParams UploadParams) (
 	return clientutils.AddProps(uploadParams.GetTargetProps(), artifactProps), nil
```

Comment:
Let's add a description for the ```addSymlinkProps``` function. Specifically, let's describe the first dtring returned value.